tasks/data_handling/adapted_gene_embeddings.py

```diff
@@ -0,0 +1,95 @@
+import scanpy as sc
+import numpy as np
+import os
+from typing import Tuple, List
+
+
+def find_common_genes(
+        folder_path: str = None,
+        save_common_genes: bool = False,
```

Comment:
This is redundant. If save_path = None, then save_common_genes = False.

Reply:
Fixed!